meilisearch-release-changelog/v1.7.0.md

```diff
@@ -0,0 +1,251 @@
+
+```bash
+[2024-02-06T14:54:11Z INFO  actix_server::builder] starting 10 workers
+```
+
+To this:
+
+```bash
+2024-02-06T13:58:14.710803Z  INFO actix_server::builder: 200: starting 10 workers
+```
+
+### Experimental: choose your log format
+
+And one new CLI parameter is introduced called `--experimental_logs_format`. This lets you specify the log format you want to use on the console.
+
+Accepted values are:
+
+- `human`: the default human-readable one
+- `json`: JSON structured logs
+
+🗣️ This feature is experimental, and we need your help to improve it! Share your thoughts and feedback on this [GitHub discussion](https://github.com/orgs/meilisearch/discussions/723).
+
+⚠️ Experimental features may be incompatible between Meilisearch versions.
+
+### Experimental: new routes to manage logs
+
+This new version of Meilisearch introduces 3 new experimental routes
+- `POST /logs/stream`: streams the log happening in real-time. Requires two parameters:
+  - `target`: defines the log level and on which part of the engine you want to apply it. It takes the form of `code_part=log_level`, for example, `index_scheduler=info`. If you want to apply the changes to the whole engine, omit `code_part=`. Valid values for log level are: `trace,` `debug,` `info,` `warn,` `error`, or `off`.
+  - `mode`: selects in what format of log you want. Two options are available: `fmt` (basic logs) or `profile` (a way more complex trace)
+- `DELETE /logs/stream`: stops the listener from the meilisearch perspective. Does not require any parameters.
+- `POST /logs/stderr`: Update the log level used in the console. It takes one parameter, `target`, which is the same as the target of the `POST /logs/stream` route.
+
+An example to disable any log from actix, but to keep all other logs at the `DEBUG` level:
+
+```bash
+curl \
+  -X POST http://localhost:7700/logs/stream \
+  -H 'Content-Type: application/json' \
+  --data-binary '{
+      "mode": "human",
+      "target": "actix=off,debug"
+    }'
+```
+
+💡 *More information in the [New experimental routes section of this file](https://meilisearch.notion.site/Logs-trace-changes-in-v1-7-0-6dd111ecd66448aba30d2f3cef212749).*
+
+⚠️ Some remarks on this `POST /logs/stream` route:
+- You can have only one listener at a time
+- Listening to the route doesn’t seem to work with `xh` or `httpie` for the moment
+- When killing the listener, it may stay installed on Meilisearch for some time, and you will need to call the `DELETE /logs/stream` route to get rid of it.
+
+🗣️ This feature is experimental, and we need your help to improve it! Share your thoughts and feedback on this [GitHub discussion](https://github.com/orgs/meilisearch/discussions/721).
+
+⚠️ Experimental features may be incompatible between Meilisearch versions.
+
+## Other improvements
+
+* Related to the [Prometheus experimental feature](https://github.com/orgs/meilisearch/discussions/625): add job variable to Grafana dashboard (#4330) @capJavert
```

Comment:
```suggestion
* [Prometheus](https://github.com/orgs/meilisearch/discussions/625): add job variable to Grafana dashboard (#4330) @capJavert
```

Reply:
I would rather have the reference to experimental feature

Reply:
```suggestion
* [Prometheus experimental feature](https://github.com/orgs/meilisearch/discussions/625): add job variable to Grafana dashboard (#4330) @capJavert
```